python/dask_cudf/dask_cudf/core.py

```diff
@@ -635,3 +635,31 @@ def reduction(
 
 def from_dask_dataframe(df):
     return df.map_partitions(cudf.from_pandas)
+
+
+for name in [
+    "add",
+    "sub",
+    "mul",
+    "truediv",
+    "floordiv",
+    "mod",
+    "pow",
+    "radd",
+    "rsub",
+    "rmul",
+    "rtruediv",
+    "rfloordiv",
+    "rmod",
+    "rpow",
+]:
+    meth = getattr(cudf.DataFrame, name)
+    DataFrame._bind_operator_method(name, meth)
+
+    meth = getattr(cudf.Series, name)
+    Series._bind_operator_method(name, meth)
+
+for name in ["lt", "gt", "le", "ge", "ne", "eq"]:
+
+    meth = getattr(cudf.Series, name)
+    Series._bind_comparison_method(name, meth)
```

Comment:
Should these binds be upstreamed into dask proper and use their abstractions on `DataFrame` and `Series`?

Reply:
Good question - We could move these binds upstream to make sure any changes in pandas-based DataFrames will also occur for cudf-based DataFrames.  However, there is currently no gpu CI in dask, so I don't think we gain much in the shor term by moving it upstream.  At the moment, it probably makes the most sense to keep this in `dask_cudf/core.py` (although I don't feel strongly about it).

Reply:
> We could move these binds upstream to make sure any changes in pandas-based DataFrames will also occur for cudf-based DataFrames. However, there is currently no gpu CI in dask, so I don't think we gain much in the shor term by moving it upstream.

I think keeping it aligned for the future makes it worth it to upstream. That being said I'm fine merging this in for a short term fix and then putting in a PR to upstream and later removing this.

Reply:
Also, even without Dask having no gpu CI, we pull the master of dask in dask-cudf CI and run nightly tests to ensure everything works as expected with the latest dask 😄

Reply:
Hmm - After looking at the dask code,  I'm thinking we either need to keep this in dask_cudf, or rewrite the way  dask is doing it upstream (this is probably what you meant by "..and use their abstractions on DataFrame and Series.").  `_bind_operator_method`/`_bind_comparison_method` are literally setting the attribute of the dask `DataFrame`/`Series` classes.  Therefore, we need to reference the `dask_cudf.DataFrame`/`Series` class definitions when binding `cudf` operators, or do this in a way that is more agnostic to the underling dataframe library.

Im sorry if I am missing an obvious solution here.  I know you may already have something elegant in mind :)